packages/plans-grid-next/src/components/features-grid/plan-storage-options.tsx

```diff
@@ -42,12 +42,6 @@ const PlanStorageOptions = ( {
 				return null;
 			}
 
-			const shouldRenderStorageTitle =
-				storageOptions.length > 0 &&
-				( storageOptions.length === 1 ||
-					intervalType !== 'yearly' ||
-					! showUpgradeableStorage ||
-					! availableForPurchase );
```

Comment:
@jeyip perhaps - do you recall scenarios where we'd not show the "Storage" title above the storage selection or the label in the grid? I'm guessing this was only needed to skip rendering it when the dropdown was rendered, which had its own label above (that we now remove completely) 🤔

Reply:
> I'm guessing this was only needed to skip rendering it when the dropdown was rendered, which had its own label above (that we now remove completely)

That seems to be the case. From what I recall there also were a few cases that we'd want to render this conditionally, but they should all be removed now.